public/app/project-setup/_components-theme.scss

```diff
@@ -0,0 +1,16 @@
+// this file contains all imports needed for components
+// but no global definitions
+@import "foundation/functions";
+
+@include exports("global") {
```

Comment:
This is a "hack" from foundation apart from their variables. 
It is a kind of cache for common includes: `global` is required from all other components, so they have made a cache to skip `global` if it was required once...